kws/model/subsampling.py

```diff
@@ -23,6 +23,14 @@ def __init__(self):
         super().__init__()
         self.subsampling_rate = 1
 
+class NoSubsampling(SubsamplingBase):
+    """No subsampling in accordance to the 'none' preprocessing
+    """
+    def __init__(self):
+        super().__init__()
```

Comment:
I think we should not add this class, because this class do noting

Reply:
this class does nothing but get the torch-script pass.
in the following if-else control, both 'linear' and 'cnn1d_s1' will be assigned a subsampling instance, while 'none' receives a None, this is forbidden by `torch-script`
  ```
  prep_type = configs['preprocessing']['type']
    if prep_type == 'linear':
        preprocessing = LinearSubsampling1(input_dim, hidden_dim)
    elif prep_type == 'cnn1d_s1':
        preprocessing = Conv1dSubsampling1(input_dim, hidden_dim)
    elif prep_type == 'none':
        preprocessing = None
    else:
        print('Unknown preprocessing type {}'.format(prep_type))
        sys.exit(1)
```